activerecord/lib/active_record/attribute_methods.rb

```diff
@@ -65,17 +65,20 @@ def eagerly_generate_alias_attribute_methods(_new_name, _old_name) # :nodoc:
       end
 
       def generate_alias_attributes # :nodoc:
-        return if @alias_attributes_mass_generated
+        klass = self
+        return if klass == ActiveRecord::Base
```

Comment:
Do we want to check for `base_class?` instead?
```suggestion
        return if base_class?
```

Similarly to Do we want to use 
```suggestion
        return if base_class?
```

https://github.com/rails/rails/blob/a0b216a8a5eab5ebb0190aebcf3628c0ae5f5399/activerecord/lib/active_record/attribute_methods.rb#L117